debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java

```diff
@@ -453,7 +453,7 @@ public static void waitForSnapshotToBeCompleted(String databaseName) {
         try {
             Awaitility.await("Snapshot not completed").atMost(Duration.ofSeconds(60)).until(() -> {
                 try {
-                    return (boolean) mbeanServer.getAttribute(getObjectName("snapshot", TEST_SERVER_NAME, databaseName), "SnapshotCompleted");
+                    return (boolean) mbeanServer.getAttribute(getObjectName("snapshot", TEST_SERVER_NAME, "0", databaseName), "SnapshotCompleted");
```

Comment:
I'd replace the magic "0" with a constant (e.g. `TEST_TASK_ID`), otherwise, it may be not clear what it is.